.groovylintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "recommended",
+    "rules": {
+        "CompileStatic": {
+            "enabled": false
+        },
+        "DuplicateStringLiteral": {
+            "enabled": false
+        },
+        "NestedBlockDepth": {
+            "enabled": false
+        }
+    }
+}
```

Comment:
Unclear to me if required, but github is upset about: 
  "no newline at end of file"